src/pybind/mgr/cephadm/module.py

```diff
@@ -683,7 +686,8 @@ def available(self) -> Tuple[bool, str]:
             return ok, err
         if not self.ssh_key or not self.ssh_pub:
             return False, 'SSH keys not set. Use `ceph cephadm set-priv-key` and `ceph cephadm set-pub-key` or `ceph cephadm generate-key`'
-        return True, ''
+
+        return True, str(self._worker_pool._processes)  # type: ignore
```

Comment:
```suggestion
        ret = {
            'workers': self._worker_pool._processes,
            'is_paused': self.paused
        }
        return True, '', ret
```

Reply:
@sebastian-philipp please review again once you get the chance. The changes requested are in and unfortunately impacted base class, cephadm, rook, the test_orchestrator and the dashboard...but we're there now.

Reply:
yeah, exactly. lgtm!